static/js/subs.js

```diff
@@ -366,6 +374,10 @@ exports.mark_sub_unsubscribed = function (sub) {
             settings.collapse('hide');
         }
 
+        // Update subscribers count
+        var count = stream_data.get_subscribers_count(sub.name) - 1;
```

Comment:
@timabbott Whenever I unsubscribe from a stream, say I am using "iago@zulip.com". When I hit unsubscribe button. And, as you advised, I iterated over sub.subscribers and I noticed "iago@zulip.com" was still there in sub.subscribers which should not be there(it vanishes from sub.subscibers after reload). 
(Values after hitting unsubscribe button)
![screenshot from 2016-03-17 11 04 18](https://cloud.githubusercontent.com/assets/6594924/13837350/6012ac8c-ec30-11e5-9a85-c985c8712a8b.png)

I think it's a bug. Only, `sub.subscribed` changes from true to false. Currently, I made some changes where I am using `sub.subscribed` to change the value of subscribers count.